Makefile.am

```diff
@@ -60,6 +60,27 @@ AM_CPPFLAGS = \
 # Utility library
 noinst_LTLIBRARIES += libeos-updater-util/libeos-updater-util-@EUU_API_VERSION@.la
 
+enum_header_files = \
+	libeos-updater-util/flatpak.h \
```

Comment:
I don’t think `flatpak.h` exists at this point in the branch yet, does it?

Reply:
Yeah, this needs to be moved to the commit that introduced flatpak support in fetch. Done.